app/ui/src/app/connections/list/list.component.scss

```diff
@@ -2,20 +2,16 @@
 
 .connections-list {
   .cards {
-    .card {
-      &.new-connection {
-        .card-pf-view {
-          background: inherit;
-          box-shadow: none;
-        }
+    ::ng-deep .card {
+      width: 25%;
+
+      &.empty-state-card {
         .card-pf-icon-large {
           color: $color-pf-blue-300;
         }
       }
```

Comment:
this should be moved to the specific component stylesheet in order to be really DRY